hieradata/role/openvpn.yaml

```diff
@@ -0,0 +1,37 @@
+---
+psick::profiles::linux_classes:
+  openvpn: openvpn
+  fw: psick::firewall::iptables
+
+
+psick::firewall::iptables::extra_rules:
+  - '-I INPUT -p tcp -m state --state NEW -m tcp --dport 1194 -j ACCEPT'
+psick::firewall::iptables::nat_rules:
+  - '-I POSTROUTING -s 10.200.200.0/24 -j MASQUERADE'
+
+psick::sysctl::settings_hash:
+  net.ipv4.ip_forward: 1
+  net.ipv4.conf.default.rp_filter: 0
+  net.ipv4.conf.all.rp_filter: 0
+  net.ipv4.conf.all.send_redirects: 0
+  net.ipv4.conf.default.send_redirects: 0
+  net.ipv4.icmp_ignore_bogus_error_responses: 1
+  net.ipv4.conf.default.log_martians: 0
+
+openvpn::servers:
+  'winterthur':
```

Comment:
winthertur here is just a name, can be anything (for example graduway), just keep the same nem here and in line 32